exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -60,384 +64,101 @@
 
 The exporter supports the following environment variables for configuration:
 
-:envvar:`OTEL_EXPORTER_ZIPKIN_ENDPOINT`: target to which the exporter will
-send data. This may include a path (e.g. http://example.com:9411/api/v2/spans).
+:envvar:`OTEL_EXPORTER_ZIPKIN_SERVICE_NAME`: Label of the local node in the
```

Comment:
I think we should use `Resource`'s `service.name`attribute from tracer provider/ span and not have a separate env for it. There have been some changes to spec around Resource and service name for exporters. tl;dr version is SDK should create a default `service.name` with `unknown_service` + (optional process executable name) if nothing provided. The env `OTEL_RESOURCE_ATTRIBUTES` with key-values pairs attributes or  `Resource` instance can be used to provided the resource related data.

Reply:
Thanks - that's new since I originally started this PR. Changing to be based off `Resource` attr `SERVICE_NAME`